### PR TITLE
use caseGroupStatus

### PR DESCRIPTION
**Pull request by tej99**
https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/236355603/US031+SEFT+Multiple+respondents+enrolled+for+the+same+RU+on+the+same+survey+view+the+same+response+status+s+Size

https://trello.com/c/1epRa0Oz/60-tbl118b-required-for-bb-display-in-my-surveys-page

DO NOT MERGE: dependent on https://github.com/ONSdigital/rm-case-service/pull/9

### CODE REVIEW

**Comment by tej99** [comment]
[TBL118b [Required for B&B] Display in My Surveys Page](https://trello.com/c/1epRa0Oz/60-tbl118b-required-for-bb-display-in-my-surveys-page)

File: frontstage_api/controllers/case_controller.py [hunk 1]
```diff
@@ -126,15 +126,14 @@ def build_full_case_data(case):
 
 def calculate_case_status(case):
     logger.debug('Getting the status of case')
-    case_events = case.get('caseEvents')
+    case_group_status = case.get('caseGroup', {}).get('caseGroupStatus')
     status = None
```
Comment (andrewmil) [inline frontstage_api/controllers/case_controller.py]: We handle the `Not Started` case status a bit strange here. 
We probably don't need to set it to None here or the if statement in the return. Instead we could just have an else statement after the `if... elif`

File: frontstage_api/controllers/case_controller.py [hunk 2]
```diff
@@ -126,17 +126,17 @@ def build_full_case_data(case):
...
-                status = 'Complete'
-                break
-            elif event['category'] == 'COLLECTION_INSTRUMENT_DOWNLOADED':
-                status = 'Downloaded'
+    case_group_status = case.get('caseGroup', {}).get('caseGroupStatus')
+
+    if case_group_status == 'COMPLETE':
+        status = 'Complete'
+    elif case_group_status == 'INPROGRESS':
+        status = 'Downloaded'
```
Comment (jwgwalton) [inline frontstage_api/controllers/case_controller.py]: Status will have a default value of "NOTSTARTED" defined at the schema level

File: frontstage_api/controllers/case_controller.py [hunk 3]
```diff
@@ -126,17 +126,17 @@ def build_full_case_data(case):
...
+    else:
+        status = 'Not Started'
```
Comment (andrewmil) [inline frontstage_api/controllers/case_controller.py]: Untested line

File: frontstage_api/controllers/case_controller.py (hunk 2 above)
Comment (tej99) [reply frontstage_api/controllers/case_controller.py]: status will be set to 'Not started' as default

**Comment by LJBabbage** [comment]
Completely outside the scope of this PR, but there are a possible 17 categories in the database and 15 would be displayed as not started to the user, which wouldn't be correct, (e.g. UNSUCCESSFUL_RESPONSE_UPLOAD). Either the categories are wrong or we need to change how we map to the user

**Comment by andrewmil** [comment]
@LJBabbage I think right now this case status is only used to decide whether the a survey is presented in the `Todo` or `History` tabs.
In the future we'll likely be giving them more information though

**Pull request merged**